A15-binary-logging.md

```diff
@@ -0,0 +1,352 @@
+# gRPC Binary Logging
+
+*   Author(s): Yuchen Zeng, Spencer Fang
+*   Approver: a11r
+*   Status: In Review
+*   Implemented in:
+*   Last updated: 07/18/2018
+*   Discussion at:
+
+## Abstract
+
+Provide binary logging functionality in gRPC.
+
+## Background
+
+gRPC binary logging logs RPCs in binary format. RPC logs generated from
+production traffic will be useful in the following cases:
+
+-   Troubleshooting services, find exceptions
```

Comment:
Grammar nit: Please make verb tenses consistent.  Change "find" to "finding".  And on line 21, change "Replay" to "Replaying".

Reply:
Done